core/src/apps/ethereum/tokens.py.mako

```diff
@@ -22,11 +24,11 @@ UNKNOWN_TOKEN = TokenInfo("Wei UNKN", 0)
 
 
 def token_by_chain_address(chain_id: int, address: bytes) -> TokenInfo:
-    if False:
+    if TYPE_CHECKING:
```

Comment:
This is wrong.

We use `if False` in this function for something else than typing.

Reply:
Reverted in 1afb0c8